UIViewController+Alert.m

```diff
@@ -32,6 +32,20 @@ - (void)showActionSheetWithTitle:(NSString *)title message:(NSString *)message o
     [self showActionSheetWithTitle:title message:message options:options handlers:handlers presentFromView:nil];
 }
 
+- (void)showiPadActionSheetWithTitle:(NSString *)title message:(NSString *)message options:(NSArray *)options handlers:(NSArray *)handlers presentFromTable:(UITableView *)presentingTable withIndexPath:(NSIndexPath *)indexPath {
```

Comment:
make the method name consistent with the existing ones - this is not the only method usable on iPad, so is `showActionSheetWithTitle:message:options:handlers:presentFromView`:

```- (void)showActionSheetWithTitle:(NSString *)title message:(NSString *)message options:(NSArray *)options handlers:(NSArray *)handlers presentFromRect:(CGRect)presentingRect inView:(UIView *)presentingView```